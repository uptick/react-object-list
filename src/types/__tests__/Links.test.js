import React from 'react'
import {
  snapshotTest,
} from 'utils/tests'
import { Links } from '../'

describe('Links type', () => {
  it('snapshot test', () => {
    const testValues = [
      {},
      {viewUrl: '/view/'},
      {editUrl: '/edit/'},
      {viewUrl: '/view/', editUrl: '/edit/'},
    ]
    testValues.forEach(testValue => {
      snapshotTest(<Links {...testValue} />)
    })
  })
})
