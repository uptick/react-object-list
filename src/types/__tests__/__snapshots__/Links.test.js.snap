// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Links type snapshot test 1`] = `
<div
  className="list-actions text-xs-right"
/>
`;

exports[`Links type snapshot test 2`] = `
<div
  className="list-actions text-xs-right"
>
  <a
    className="btn btn-sm btn-link"
    href="/view/"
  >
    View
  </a>
</div>
`;

exports[`Links type snapshot test 3`] = `
<div
  className="list-actions text-xs-right"
>
  <a
    className="btn btn-sm btn-link"
    href="/edit/"
  >
    Edit
  </a>
</div>
`;

exports[`Links type snapshot test 4`] = `
<div
  className="list-actions text-xs-right"
>
  <a
    className="btn btn-sm btn-link"
    href="/view/"
  >
    View
  </a>
  <span>
    Â 
  </span>
  <a
    className="btn btn-sm btn-link"
    href="/edit/"
  >
    Edit
  </a>
</div>
`;
